### DISCUSSION TITLE
"Create & Add Another" not hideable/showing unauthorized

### DISCUSSION THREAD

**Discussion by Muetze42** [laravel/nova-issues#6125] [Ideas & Feature Requests]
- Laravel Version: 10.38.1
- Nova Version: 4.32.9

### Description:

I limit the models that can be created:
```php
    /**
     * Determine whether the user can create models.
     */
    public function create(Model $user): bool
    {
        return Model::count < 5;
    }
```

If I'm in the Create form of the 5 model, then the `Create & Add Another` button should no longer be shown.

But it is still shown and I have not found a way to deactivate it manually.

**Comment by sanderbaas** [comment]
@Muetze42  Did you ever find a way to hide the `Create & Add Another` button?
